### PR TITLE
Bluehost on Plans: Feature refresh for Bluehost launch

### PR DESCRIPTION
**Pull request by ddc22**
Related to #

## Proposed Changes

* Adds the feature enhancements done for Bluehost based landing pages to Calypso
* Document: https://docs.google.com/document/d/1gqQWVtccPckyOsJv9rLQ9yvIRYOezWQSoQzBHVJG5Rg/edit

image

## Testing Instructions

* Go to  `/start`
* Reach plans step
* Make sure all the suggested feature changes and tooltip changes are visible

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Comment by ddc22** [comment]
@niranjan-uma-shankar if you have some time can you go over this to make sure everything is in order?

**Review by oswian — COMMENTED** [review]
Just noting a few differences between the site and doc:

- Free plan: Missing "Access to ..." at beginning of "Dozens of premium themes"
- Creator plan: "Live chat support" is ordered above "No limitations on site visitors"
- Creator plan: "2% transaction fee ..." is ordered above "0% transaction fee ..."

**Comment by niranjan-uma-shankar** [comment]
> @niranjan-uma-shankar if you have some time can you go over this to make sure everything is in order?

@porada Could you please help test and review this PR for @ddc22 ?

Please also check the conversation in p1710137480228109?thread_ts=1709671065.838889-slack-C06FCT6EEHH about a potential experiment in Calypso, which if confirmed means that the Bluehost change might not be the first thing we launch.

File: client/lib/plans/features-list.tsx [hunk 1]
```diff
@@ -2044,9 +2060,25 @@ export const FEATURES_LIST: FeatureList = {
...
+		getTitle: () => i18n.translate( 'Vulnerability notifications for core and plugins' ),
+		getDescription: () =>
+			i18n.translate(
+				'Stress less knowing there’s a dedicated team identifying potential vulnerabilities for WordPress and plugins, ensuring early detection and preventing future attacks.'
+			),
+	},
 	[ FEATURE_SECURITY_DDOS ]: {
 		getSlug: () => FEATURE_SECURITY_DDOS,
-		getTitle: () => i18n.translate( 'DDOS mitigation' ),
+		getTitle: () => i18n.translate( 'DDoS protection and mitigation' ),
```
Comment (porada) [inline client/lib/plans/features-list.tsx]: The marketing page use `'DDOS protection and mitigation'`, which has already been translated. Maybe we could do the equivalent of `fixme__( 'DDoS protection and mitigation', __( 'DDOS protection and mitigation' ) )` here.

**Comment by porada** [comment]
Apart from that one comment, the changes look consistent with the marketing pages 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/11595520

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @ddc22 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/11595520).

**Comment by ddc22** [comment]
@niranjan-uma-shankar @porada I guess we need to put this on hold for now?

**Comment by niranjan-uma-shankar** [comment]
> I guess we need to put this on hold for now?

Sorry if I missed any context, but what's the reason to put this on hold? I think the Akismet feature is missing, and once we have that included we can get this shipped?

**Comment by porada** [comment]
I’ll merge this as-is (after it’s rebased) and handle Akismet in a separate PR.

**Comment by niranjan-uma-shankar** [comment]
I'm reverting the PR since the new strings are not translated https://github.com/Automattic/wp-calypso/pull/89231. That's strange because the `String Freeze` label was added three weeks ago, so translations should've completed by now.

**Comment by niranjan-uma-shankar** [comment]
**UPDATE**

I'll hold back on the revert until I hear from Team Global p1712297340172389-Slack-C02AED43D.

**Pull request merged**